.github/workflows/promote.yaml

```diff
@@ -0,0 +1,42 @@
+name: Promote Charm
+
+on:
+  workflow_dispatch:
+    inputs:
+      promotion:
+        type: choice
+        description: Channel to promote from
+        options:
+          - edge -> beta
+          - beta -> candidate
+          - candidate -> stable
+
+jobs:
+  promote:
+    name: Promote Charm
+    runs-on: ubuntu-latest
```

Comment:
We use Ubuntu 20.04 in all other places, including `charmcraft.yaml`. Don't we want to use the same version here?

Reply:
✅